lib/arclight/custom_document.rb

```diff
@@ -31,5 +34,34 @@ def to_solr(solr_doc = {})
     def names
       [corpname, famname, name, persname].flatten.compact.uniq - repository
     end
+
+    # Combine subjets into one group from:
+    #  <controlaccess/><subject></subject>
+    #  <controlaccess/><function></function>
+    #  <controlaccess/><genreform></genreform>
+    #  <controlaccess/><occupation></occupation>
+    def access_subjects(subjects = [])
```

Comment:
@jkeck - Not that I know of. I was followed the NYU Library code: https://github.com/NYULibraries/ead_indexer/blob/master/lib/ead_indexer/behaviors.rb#L55-L58 which initializes the subject variable as a param to the def.

Would you like me to change that?